webapp-src/src/Login/GrantScope.js

```diff
@@ -112,12 +122,12 @@ class GrantScope extends Component {
           <button type="button" className="btn btn-primary" onClick={this.handleGrantScope}>{i18next.t("login.grant")}</button>
         </div>
       </div>
-      <hr/>
+      <hr style={{borderTop: "0px"}}/>
```

Comment:
As mentionned in [React doc](https://en.reactjs.org/docs/dom-elements.html#style), `style` in react should be avoided, I'd use css instead

Reply:
Well it should be avoided everywhere. Sorry about that… Bad habit from childhood.

Reply:
done